app/2.0/docs/tools/create-app-polymer-cli.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Create an application project with the Polymer CLI
+---
+
+<!-- toc -->
+
+Polymer CLI supports initializing a project folder with one of several application templates.  
+The `polymer-2-application` template is the most basic starting point for a Polymer-based 
+application. Other templates introduce more complex layout and application patterns.
+
+This chapter teaches you more about the `polymer-2-application` template.  
+See [Polymer App Toolbox templates](../../toolbox/templates) for more details on other templates.
+
+## App project architecture {#app-architecture}
+
+The `polymer-2-application` template follows the [app shell 
+architecture](https://developers.google.com/web/updates/2015/11/app-shell) pattern.
```

Comment:
I think this whole section is incorrect, or at least misleading. The `polymer-2-application` template doesn't really have any structure at all. The PSK template follows the app shell architecture.